tests/features/bootstrap/FeatureContext.php

```diff
@@ -1109,7 +1110,41 @@ public function assertPageNotCached() {
    *
    * @Given I select/check the :row_text row
    */
-  public function assertSelectRow($text) {
+  public function assertSelectRow(string $text): void {
+    $checkbox = $this->getRowCheckboxByText($text);
+    $checkbox->check();
+  }
+
+  /**
+   * Attempts to uncheck a checkbox in a table row containing a given text.
+   *
+   * @param string $text
+   *   Text in the row.
+   *
+   * @Given I deselect/uncheck the :row_text row
+   */
+  public function assertDeselectRow(string $text): void {
```

Comment:
This is not an assertion, so let's rename the method to something that makes more sense like `deselectNamedRow()`.